tests/providers/google/cloud/hooks/test_pubsub.py

```diff
@@ -384,13 +395,14 @@ def test_publish_api_call_error(self, mock_service):
         publish_method.side_effect = GoogleAPICallError(f"Error publishing to topic {EXPANDED_SUBSCRIPTION}")
 
         with pytest.raises(PubSubException):
-            self.pubsub_hook.publish(project_id=TEST_PROJECT, topic=TEST_TOPIC, messages=TEST_MESSAGES)
+            self.pubsub_hook.publish(project_id=TEST_PROJECT, topic=TEST_TOPIC, messages=[{"data": b"test"}])
```

Comment:
Since the publish function throws an exception regardless of the value of `messages`, there was no need to continue using the global variable `TEST_MESSAGES`.